Vagrantfile

```diff
@@ -30,6 +30,11 @@ Vagrant::configure("2") do |config|
     vb.customize ["modifyvm", :id, "--memory", BOX_MEMORY]
   end
 
+  config.vm.provider :vmware_fusion do |vbox, override|
+    vbox.memory = 1024
+    override.vm.box = "phusion/ubuntu-14.04-amd64"
```

Comment:
@sglebs did you intent to include the Vagrantfile in this PR? Seems non-sequitur to the PR title...